cli/run.go

```diff
@@ -21,14 +21,13 @@ import (
 
 func init() {
 	cmd := register("run", runRun, `
-usage: flynn run [-d] [-r <release>] [-e <entrypoint>] [-l] [--] <command> [<argument>...]
+usage: flynn run [-d] [-r <release>] [-l] [--] <command> [<argument>...]
 
 Run a job.
 
 Options:
 	-d, --detached    run job without connecting io streams (implies --enable-log)
 	-r <release>      id of release to run (defaults to current app release)
-	-e <entrypoint>   overwrite the default entrypoint of the release's image
```

Comment:
Instead of deleting the flag right away, we should print a deprecation message and use it as the first flag.